ci/travis-after-success.sh

```diff
@@ -1,6 +1,9 @@
 #!/bin/sh
 set -xe
 
+# Submit code coverage report
+curl -s https://codecov.io/bash | bash
```

Comment:
Do you consider "navigating through 1500 lines of convoluted bash code to hacking own replacement" a viable alternative? This is the only "official" way.

To be honest I don't see much harm in it as it is a public project tested in vitual environment - what can possibly happen?